apps/api/src/graphql/resolvers/notification.ts

```
@@ -0,0 +1,69 @@
+import type { AnnouncementNotificationData, CommentNotificationData } from '@/db/schemas/json';
+
+Notification.implement({
+  fields: (t) => ({
+    id: t.exposeID('id'),
+    state: t.expose('state', { type: NotificationState }),
+    category: t.expose('category', { type: NotificationCategory }),
+    data: t.expose('data', { type: NotificationData }),
+  }),
+});
+
+const CommentNotificationData = builder.objectRef<CommentNotificationData>('CommentNotificationData').implement({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'CommentNotificationData'. Consider to delete it or rename it.

'CommentNotificationData' is defined here:


(lint/suspicious/noRedeclare)